pkg/tests/ossm/smoke_test.go

```diff
@@ -15,55 +15,40 @@
 package ossm
 
 import (
-	"strings"
+	"net/http"
 	"testing"
-	"time"
 
 	"github.com/maistra/maistra-test-tool/pkg/app"
-	. "github.com/maistra/maistra-test-tool/pkg/examples"
-	. "github.com/maistra/maistra-test-tool/pkg/util"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/require"
+	"github.com/maistra/maistra-test-tool/pkg/util/curl"
 	. "github.com/maistra/maistra-test-tool/pkg/util/log"
-	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
 	. "github.com/maistra/maistra-test-tool/pkg/util/test"
 )
 
-func cleanupBookinfo() {
-	Log.Info("Cleanup")
-	app := Bookinfo{"bookinfo"}
-	app.Uninstall()
-	time.Sleep(time.Duration(30) * time.Second)
-}
-
 func TestBookinfo(t *testing.T) {
 	NewTest(t).Id("A2").Groups(ARM, Full, Smoke, InterOp).Run(func(t TestHelper) {
-		Log.Info("Test Bookinfo Installation")
-		app.InstallAndWaitReady(t, app.Bookinfo("bookinfo"))
 
-		Log.Info("Check pods running 2/2 ready")
-		msg, _ := Shell(`oc get pods -n bookinfo`)
-		if strings.Contains(msg, "2/2") {
-			Log.Info("Success. proxy container is running.")
-		} else {
-			t.Error("Error. proxy container is not running.")
-		}
```

Comment:
We should keep this check, but rewrite it to use one of the new helpers. But this check was also not quite correct. It checked if at least one pod is injected. When you rewrite this, please check that the output contains `2/2` and also that it does not contain any `1/1`.  We could also perform this check for every bookinfo pod, so that we're sure that all pods are injected (if we do this, we don't need to check for `1/1`).